python/lsst/ts/criopy/m1m3/slew_controller_page_widget.py

```diff
@@ -170,3 +154,24 @@ def __init__(self, m1m3: MetaSAL):
         layout.addWidget(ChartWidget(velocity_axis, max_items=50 * 5))
 
         self.setLayout(layout)
+
+        self.m1m3.detailedState.connect(self.detailed_state)
+        self.m1m3.forceControllerState.connect(self.force_controller_state)
+
+    def __set_enabled(self) -> None:
+        self.setEnabled(self.__active_state is True and self.__slew_flag is False)
+
+    @Slot()
+    def detailed_state(self, data: typing.Any) -> None:
```

Comment:
If this `data` comes from the SAL, it should be `salobj.BaseMsgType`. Otherwise, `types.SimpleNamespace` for an `object`.

Reply:
Was changed there and everywhere else.